app/controllers/health_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class HealthController < ApplicationController
+  rescue_from(Exception) { render_down }
+
+  def show
+    render_up
+  end
+
+  private
+
+  def render_up
+    render html: html_status(color: 'green')
+  end
+
+  def render_down
+    render html: html_status(color: 'red'), status: :internal_server_error
+  end
+
+  # rubocop:disable Rails/OutputSafety
+  def html_status(color:)
+    %(<html><body style="background-color: #{color}"></body></html>).html_safe
+  end
```

Comment:
@Tuubz Since we are developing API application so do you think we should return `json` as default instead? 😄 

```
    def render_up
        render json: { status: :ok }
    end

    def render_down
      render json: { status: :internal_server_error }
    end
```

Reply:
That's not for API consumers but you can make use of `respond_to` with different formats (see example: https://guides.rubyonrails.org/action_controller_overview.html#rendering-xml-and-json-data):

```
def show
  respond_to do |format|
      format.html  { render_up }
      format.json { render json: {}, status: :ok}
    end
end
```

Reply:
@olivierobert I tried with `respond_to` to support `html` and `json`, it works on the browser when I visit `localhost:3000`, when trying on Postman it always returns the response as `html` format. I found a suggestion about adding `.json` suffix to the endpoint but not sure how it work with the root. Am I missing something? 🤔 

![image](https://github.com/nimblehq/ic-rails-huey-toby/assets/6950766/98689c81-b27d-4953-b0a1-67d29c3930ef)

![image](https://github.com/nimblehq/ic-rails-huey-toby/assets/6950766/a6f87c1d-dd54-459c-baeb-217fe5836274)

Reply:
I believe you are missing a second header `Accept: application/json` in addition to `Content-Type`

See https://webmasters.stackexchange.com/questions/31212/difference-between-the-accept-and-content-type-http-headers

Welcome to the Web and its intricacies 🔬

Reply:
@olivierobert Niceeee! 🎉 Finally it works 🚀 cc @Tuubz